spanner/api/Spanner.Samples.Tests/CreateInstanceConfigAsyncTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Spanner.Samples.Tests
+{
+
+    [Collection(nameof(SpannerFixture))]
+    public class CreateInstanceConfigAsyncTest
+    {
+        private readonly SpannerFixture _spannerFixture;
+
+        public CreateInstanceConfigAsyncTest(SpannerFixture spannerFixture)
+        {
+            _spannerFixture = spannerFixture;
+        }
+
+        [Fact]
+        public async Task TestCreateInstanceConfigAsync()
+        {
+            CreateInstanceConfigAsyncSample createInstanceConfigAsyncSample = new CreateInstanceConfigAsyncSample();
+
+            var instanceConfig = await createInstanceConfigAsyncSample.CreateInstanceConfigAsync(_spannerFixture.ProjectId,
+                _spannerFixture.InstanceConfigId, _spannerFixture.CustomInstanceConfigId);
```

Comment:
Similar comment here, if this test executes last, then update and delete won't be able to do anything.
I think it's fine if we just have independen configs for each test.

Or is there something I'm missing.